src/components/informationCard/InformationCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './InformationCard.css';
+import {ArrowRight} from "@phosphor-icons/react";
+
+const InformationCard = ({infoCardTitle, infoCardContent, experienceNeeded, onClick}) => {
+    return (
+        <div className="infoCard">
+            <h3 className="info-card-title">{infoCardTitle}</h3>
+            <p className="info-card-content">{infoCardContent}</p>
+            {experienceNeeded && <p className="info-card-experience-tag"><em>Enige IT-ervaring vereist</em></p>}
+            <button
+                type="button"
+                onClick={onClick}
+                className="arrow-button"
```

Comment:
We hebben meer buttons op de website die je, als je deze css-class ziet, zou kunnen aanduiden als arrow-button. Dus probeer 'm nog ietsje specifieker te beschrijven, zoals 'round-arrow-button` o.i.d.